proposals/low-level-struct-improvements.md

```diff
@@ -219,7 +219,11 @@ Span<int> BadUseExamples(int parameter)
 
 The `scoped` annotation also means that the `this` parameter of a `struct` can now be defined as `scoped ref T`. Previously it had to be special cased in the rules as `ref` parameter that had different *ref-safe-to-escape* rules than other `ref` parameters (see all the references to including or excluding the receiver in the span safety rules). Now it can be expressed as a general concept throughout the rules which further simplifies them.
 
-In addition to parameters the `scoped` annotation can be applied to locals or `struct` instance methods. These annotations have the same impact to lifetimes when applied to locals. For locals these annotations concretely define the lifetimes and override the lifetime that would be inferred via the initializer. 
+The `scoped` annotation can also be applied to the following locations:
+
+- locals: This annotation sets the lifetime as *safe-to-escape*, or *ref-safe-to-escape* in case of a `ref` local, to of *current method* irrespective of the initializer lifetime. 
+- `struct` instance methods: This annotation sets the type of `this` to `scoped ref T`. This is the default value but it is useful in the case the overall `struct` type is declared as `unscoped` (detailed [below](#return-fields-by-ref)).
```

Comment:
Is this still needed now that API review has disallowed adding `[UnscopedRef]` to a whole type? 🤔
See: https://github.com/dotnet/runtime/issues/72074#issuecomment-1191764092.